zinit.zsh

```diff
@@ -3232,4 +3232,9 @@ if [[ -e ${${ZINIT[BIN_DIR]}}/zmodules/Src/zdharma/zplugin.so ]] {
 
 # Create so that for sure no warncreateglobal warning is issued
 typeset -g REPLY
+
+# Add $ZPFX/lib/pkg-config to PKG_MODULE_DIR, so that libraries
+# installed locally can be found by autotools and cmake
+export PKG_CONFIG_PATH=$ZPFX/lib/pkgconfig:$PKG_CONFIG_PATH
```

Comment:
Does it make sense to add double quotes?

```suggestion
export PKG_CONFIG_PATH="$ZPFX/lib/pkgconfig":"$PKG_CONFIG_PATH"
```

Reply:
Put `"""`

Reply:
> Does it make sense to add double quotes?
> 
> ```suggestion
> export PKG_CONFIG_PATH="$ZPFX/lib/pkgconfig":"$PKG_CONFIG_PATH"
> ```

Yes but only against that weird option that makes all variables as active globs carriers. I'll add them I'm currently on phone.